src/views/LimitedRestorationToFull.vue

```diff
@@ -308,12 +308,12 @@ export default class LimitedRestorationToFull extends Vue {
       deliveryAddress: applicant.deliveryAddress,
       mailingAddress: applicant.mailingAddress,
       officer: {
-        email: applicant.officer.email,
-        firstName: applicant.officer.firstName,
-        lastName: applicant.officer.lastName,
-        middleName: applicant.officer.middleName,
-        organizationName: applicant.officer.organizationName,
-        partyType: applicant.officer.partyType,
+        email: applicant?.officer.email,
+        firstName: applicant?.officer.firstName,
+        lastName: applicant?.officer.lastName,
+        middleName: applicant?.officer.middleName,
+        organizationName: applicant?.officer.organizationName,
+        partyType: applicant?.officer.partyType,
```

Comment:
Shouldn't need to do this since `applicant` was checked on line 261.

If the intent is to protect this method from bad data then applicant should be checked directly, maybe on line 306.

If we must use conditional chaining here then it should be done lines 308, 309 and 319 as well (but my vote is we don't need CC here).

Reply:
Reverted